admin/app/controllers/admin/CommercialController.scala

```diff
@@ -44,7 +45,9 @@ class CommercialController(
 
   def renderSpecialAdUnits: Action[AnyContent] =
     Action { implicit request =>
-      val specialAdUnits = dfpApi.readSpecialAdUnits(Configuration.commercial.dfpAdUnitGuRoot)
+      val specialAdUnits =
+        if (LineItemJobs.isSwitchedOn) { Store.getDfpSpecialAdUnits }
+        else { dfpApi.readSpecialAdUnits(Configuration.commercial.dfpAdUnitGuRoot) }
       Ok(views.html.commercial.specialAdUnits(specialAdUnits))
```

Comment:
I can say many of the rendering methods have NoCache so I wonder why this might be different and if we need to change that after writing to an S3 🤔 @Jakeii

Reply:
I believe because we want to get the most recent data from the S3. Here 42b3db935cea4cae58a2040991220e4641fe6348. I will resolve this conversation and merge.